### ISSUE TITLE
[Feature Request] FileFlows widget

### ISSUE THREAD

**Issue by Mbarmem** [benphelps/homepage#605]
### Description

Could you make a widget for [FileFlows](https://fileflows.com/)?

[Api](https://docs.fileflows.com/api-help#/) info can be found here.

Eg: [/api/status](https://docs.fileflows.com/api-help#/Status/get_api_status)

```
{
  "queue": 0,
  "processing": 0,
  "processed": 0,
  "time": "string"
}
```

**Comment by pcking99** [comment]
I would love to see this supported too.

**Comment by Mbarmem** [comment]
Closed #990
